ui/apps/platform/package.json

```diff
@@ -99,7 +99,6 @@
         "typeface-open-sans": "^1.1.13",
         "typeface-open-sans-condensed": "^1.1.13",
         "use-deep-compare-effect": "^1.8.1",
-        "victory-core": "^36.2.1",
```

Comment:
This was needed solely to import prop types that were not re-exported from PatternFly's React charts library. The update introduced a type error, so I took the opportunity to remove the reliance on this nested dependency.